### PR TITLE
Create transport-layer-security.md

### LINKED ISSUE #2579: [hi] Localize Transport Layer Security for Hindi
I would like to work on this!

### PR DESCRIPTION
**Pull request by Aryan-Singla**
### Describe your changes
Localize Transport Layer Security for Hindi

### Related issue number or link (ex: `resolves #issue-number`)
resolves #2579

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by kumarankit999 — COMMENTED** [review]
lgtm

**Comment by seokho-son** [comment]
@ARYANis0001 I have a common comment on your PRs.
PTAL: https://github.com/cncf/glossary/pull/2605#issuecomment-1815774999

**Comment by seokho-son** [comment]
Hi @kumarankit999 
I think the content of this PR is not visible in the preview, not approvable as is. (the reason why I don't mention how to fix is that @ARYANis0001 already had simmir request to fix in related PRs)
Anyway please try to check the Preview as well when you review PR. :)

**Comment by kumarankit999** [comment]
Ok @seokho-son , I will keep in my mind!

**Comment by ARYANis0001** [comment]
> Ok @seokho-son , I will keep in my mind!

Hey @seokho-son  
Hey @kumarankit999 
I want to express my sincere apologies for any inconvenience caused to the team members of CNCF/Glossary due to my raised PRs. I fully understand my mistake and acknowledge that I need to be more careful in the future. Going forward, I will ensure to thoroughly read the contributing guidelines and submit PRs in my native language, which I am fluent in and proficient in.

**Comment by ARYANis0001** [comment]
> @ARYANis0001 I have a common comment on your PRs.
> PTAL: https://github.com/cncf/glossary/pull/2605#issuecomment-1815774999

Hey @seokho-son  
Hey @kumarankit999 
I want to express my sincere apologies for any inconvenience caused to the team members of CNCF/Glossary due to my raised PRs. I fully understand my mistake and acknowledge that I need to be more careful in the future. Going forward, I will ensure to thoroughly read the contributing guidelines and submit PRs in my native language, which I am fluent in and proficient in.

**Comment by seokho-son** [comment]
Hello dev-hi team ! :)

https://github.com/cncf/glossary/pull/2605#issuecomment-1818151180

I think @ARYANis0001 closed other PRs except PRs for Hindi localization. I guess @ARYANis0001 is now focusing on Hindi localization only. Please guide @ARYANis0001

**Comment by ARYANis0001** [comment]
@seokho-son 
Yes I will now only contribute to Hindi localisation because Hindi is my native language and I am very proficient and fluent in Hindi

**Comment by kumarankit999** [comment]
Yes @seokho-son now @ARYANis0001 has understood and will be able to do amazing contributions here!!

**Comment by ARYANis0001** [comment]
Hey @aj11anuj @kumarankit999 @bishal7679 
I have added some words to simplify it can you please review it

**Comment by Aryan-Singla** [comment]
Hey @aj11anuj @kumarankit999  please review it

File: content/hi/transport-layer-security.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: ट्रांसपोर्ट लेयर सिक्योरिटी (TLS)
+status: पूर्ण
```
Comment (seokho-son) [inline content/hi/transport-layer-security.md:3]: the value for status should not be localized. 
only valid english status is acceptable. I am sure the work in this PR is not visible.

Comment (Aryan-Singla) [reply content/hi/transport-layer-security.md]: I have already added some missing words from my side around 2 weeks ago commit (https://github.com/cncf/glossary/pull/2593/commits/866d77e30d713c6d4749067a890230b8d93f9cf9). However, the status localization was skipped by me by mistake. Now this PR is pending for merge. If any other helpful suggestion is encountered by me or by anyone then I will update it

**Comment by Aryan-Singla** [comment]
Hey @Garima-Negi Can you please review it

File: content/hi/transport-layer-security.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: ट्रांसपोर्ट लेयर सिक्योरिटी Transport Layer Security (TLS)
```
Comment (aj11anuj) [inline content/hi/transport-layer-security.md:2]:
```suggestion
title: ट्रांसपोर्ट लेयर सिक्योरिटी (Transport Layer Security)
```

File: content/hi/transport-layer-security.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+यह इंटरनेट पर भेजे गए डेटा की सुरक्षित डिलीवरी सुनिश्चित करता है,
+जैसे डेटा की संभावित निगरानी और/या परिवर्तन से बचानाा।
+यह प्रोटोकॉल मैसेजिंग, ई-मेल आदि जैसे अनुप्रयोगों में व्यापक रूप से उपयोग किया जाता है।
+
+## यह किस समस्या का समाधान करता है
+
+टीएलएस के बिना, ब्राउज़िंग आदतें, ई-मेल पत्राचार, ऑनलाइन चैट और कॉन्फ्रेंसिंग कॉल जैसी संवेदनशील जानकारी
+ट्रांसमिशन के दौरान दूसरों द्वारा आसानी से पता लगाया और संशोधित किया जा सकता है।
+टीएलएस का समर्थन करने के लिए सर्वर और क्लाइंट एप्लिकेशन को सक्षम करना यह सुनिश्चित करता है
+उनके बीच प्रसारित डेटा को गोपित रखता है और डेटा तीसरे पक्ष द्वारा देखने योग्य नहीं है।
```
Comment (aj11anuj) [inline content/hi/transport-layer-security.md:20]: Reading fluency of these phrases are not matching with each other.

**Review by kumarankit999 — COMMENTED** [review]
looks nice now!

File: content/hi/transport-layer-security.md (hunk 3 above)
Comment (Aryan-Singla) [reply content/hi/transport-layer-security.md:20]: yes not matching any suggestions how to solve

Comment (Aryan-Singla) [reply content/hi/transport-layer-security.md:20]: It try to match them

Comment (Aryan-Singla) [reply content/hi/transport-layer-security.md]: I have update the lines

**Comment by Aryan-Singla** [comment]
Hello @aj11anuj if you have bandwidth then Please check this localization is there any need for improvement

File: content/hi/transport-layer-security.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+status: Completed
+category: अवधारणा
+tags: ["सिक्योरिटी", "नेटवर्किंग"]
+---
+
+## यह क्या है
+
+ट्रांसपोर्ट लेयर सिक्योरिटी (टीएलएस) एक प्रोटोकॉल है जिसे नेटवर्क पर संचार को बढ़ी हुई सुरक्षा प्रदान करने के लिए डिज़ाइन किया गया है।
+यह इंटरनेट पर भेजे गए डेटा की सुरक्षित डिलीवरी सुनिश्चित करता है,
+जैसे डेटा की संभावित निगरानी और/या परिवर्तन से बचानाा।
```
Comment (aj11anuj) [inline content/hi/transport-layer-security.md:12]:
```suggestion
जैसे डेटा की संभावित निगरानी या परिवर्तन से बचाव।
```

File: content/hi/transport-layer-security.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+यह प्रोटोकॉल मैसेजिंग, ई-मेल आदि जैसे अनुप्रयोगों में व्यापक रूप से उपयोग किया जाता है।
+
+## यह किस समस्या का समाधान करता है
+
+टीएलएस के बिना, ब्राउज़िंग आदतें, ई-मेल पत्राचार, ऑनलाइन चैट और कॉन्फ्रेंसिंग कॉल जैसी संवेदनशील जानकारी
```
Comment (aj11anuj) [inline content/hi/transport-layer-security.md:17]:
```suggestion
टीएलएस के बिना, ब्राउज़िंग आदतें, ई-मेल पत्रचार, ऑनलाइन चैट और कॉन्फ्रेंसिंग कॉल जैसी संवेदनशील जानकारी
```

File: content/hi/transport-layer-security.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+ट्रांसमिशन के दौरान दूसरों द्वारा आसानी से पता लगाया और संशोधित किया जा सकता है।
```
Comment (aj11anuj) [inline content/hi/transport-layer-security.md:18]:
```suggestion
ट्रांसमिशन के दौरान दूसरों द्वारा आसानी से पता लगाई और संशोधित कि जा सकती है।
```

File: content/hi/transport-layer-security.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+टीएलएस का समर्थन करने के लिए सर्वर और क्लाइंट एप्लिकेशन को सक्षम या इनेबल करना यह सुनिश्चित करता है
```
Comment (aj11anuj) [inline content/hi/transport-layer-security.md:19]:
```suggestion
टीएलएस का समर्थन करने के लिए सर्वर और क्लाइंट एप्लिकेशन को सक्षम या इनेबल करना यह सुनिश्चित करता है,
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
.

**Review by aj11anuj — APPROVED** [review]
Lgtm

**Comment by aj11anuj** [comment]
@kumarankit999

**Review by abhay-raj19 — APPROVED** [review]
Looks Good to me 🚀

**Comment by abhay-raj19** [comment]
Thanks For contributing ..Looking forward for more such !!🙂

**Pull request merged**